README.md

```diff
@@ -2,23 +2,75 @@
 [![Build Status](https://api.travis-ci.org/kubernetes-incubator/cri-containerd.svg?style=flat-square)](https://travis-ci.org/kubernetes-incubator/cri-containerd)
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-incubator/cri-containerd?style=flat-square)](https://goreportcard.com/report/github.com/kubernetes-incubator/cri-containerd)
 
-cri-containerd is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
+`cri-containerd` is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
 ![cri-containerd](./docs/cri-containerd.png)
 ## Current Status
-We are currently working toward an alpha version to be used in conjunction with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones) for information about current and future milestones.
+The current release of `cri-containerd` is `v0.1.0-alpha.1`. This release is for
```

Comment:
cri-containerd is in Alpha. It works with Kubernetes 1.7+. See the roadmap...

Mentioned in https://github.com/kubernetes-incubator/cri-containerd/pull/89#discussion_r124402843.